docs/guide/architecture-styles/n-tier-content.md

```diff
@@ -4,7 +4,7 @@ An N-tier architecture divides an application into **logical layers** and **phys
 
 Layers are a way to separate responsibilities and manage dependencies. Each layer has a specific responsibility. A higher layer can use services in a lower layer, but not the other way around.
 
-Tiers are physically separated, running on separate machines. A tier can call to another tier directly, or use asynchronous messaging (message queue). Although each layer might be hosted in its own tier, that's not required. Several layers might be hosted on the same tier. Physically separating the tiers improves scalability and resiliency, but also adds latency from the additional network communication.
+Tiers are physically separated, running on separate machines. A tier can call to another tier directly, or use [asynchronous messaging (message queue)](/previous-versions/msp-n-p/dn589781(v=pandp.10)). Although each layer might be hosted in its own tier, that's not required. Several layers might be hosted on the same tier. Physically separating the tiers improves scalability and resiliency, but also adds latency from the additional network communication.
```

Comment:
We don't create new links to the archived content anymore. Can you instead please link to active content?

Reply:
OK - it's just I saw that page being linked to from other places, like:

https://learn.microsoft.com/en-us/azure/architecture/patterns/queue-based-load-leveling

It could be quite a bit more work to pull out and restructure the content from the [Asynchronous Messaging Primer](https://learn.microsoft.com/en-us/previous-versions/msp-n-p/dn589781(v=pandp.10)) though. 

Do you have that work already planned and/or underway?
That way, I could join and collaborate on that without having to lay all the groundwork myself.

Reply:
We do not have that work scheduled.  And editorial policy says no additional linking.  So we need to find an alternative on site to link to or have another article created if there is a gap.